__tests__/endpoints.test.js

```diff
@@ -103,11 +107,88 @@ describe('GET /api/articles/:article_id', () => {
         return request(app)
         .get('/api/articles/999')
         .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe('article does not exist');
+        });
     });
     test('should return a 400 status code when passed an invalid article number', () => {
         return request(app)
         .get('/api/articles/pies')
         .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('Bad request');
+        });
     });
 });
-    
\ No newline at end of file
+    
+describe.only('GET /api/articles', () => {
+    test('should return a 200 status code ', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+    });
+    test('should respond with articles with the following keys', () => {
```

Comment:
We wouldn't really need this test since your next test will fail if it doesn't have these properties.